<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnterResult</class>
 <widget class="QDialog" name="EnterResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLineEdit" name="txtValue"/>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
